test/functional/net.py

```diff
@@ -49,6 +50,15 @@ def run_test(self):
         assert_equal(self.nodes[0].getnetworkinfo()['networkactive'], True)
         assert_equal(self.nodes[0].getnetworkinfo()['connections'], 2)
 
+        # test getaddednodeinfo
+        # add a node (node2) to node0
+        ip_port = "127.0.0.1:{}".format(p2p_port(2))
+        peers = self.nodes[0].addnode(ip_port, 'add')
```

Comment:
nit: no need to assign this unused `peers` variable.

I recommend you run flake8 over PRs to catch nits like this.